### PR TITLE
MM-11684 Fix grey highlight in the text fields in user profile

### PR DESCRIPTION
**Pull request by sudheerDev**
#### Summary
Fixes grey highlight in text fields

#### Ticket Link
[MM-11684](https://mattermost.atlassian.net/browse/MM-11684)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Has UI changes

#### Device Information
This PR was tested on: [Iphone x simulator] 

#### Screenshots
Before:
screen shot 2018-08-27 at 11 17 46 pm

After:
screen shot 2018-08-27 at 11 18 11 pm

### CODE REVIEW

File: app/components/search_bar/search_box.js [hunk 1]
```diff
@@ -339,45 +339,46 @@ export default class Search extends Component {
...
-                            width: this.inputFocusWidthAnimated,
-                            paddingLeft: this.inputFocusPlaceholderAnimated,
-                        },
-                        this.props.shadowVisible && {
-                            shadowOffset: {width: this.props.shadowOffsetWidth, height: this.shadowHeight},
-                            shadowColor: this.props.shadowColor,
-                            shadowOpacity: this.shadowOpacityAnimated,
-                            shadowRadius: this.props.shadowRadius,
-                        },
+                <View style={this.props.inputStyle}>
```
Comment (sudheerDev) [inline app/components/search_bar/search_box.js]: Should I change this prop name? i am 0/5 in this so left it this way. Lemme know if anyone prefers something like `inputWrapperStyle`

**Comment by sudheerDev** [comment]
@enahum Changed it

File: app/components/search_bar/search_box.js (hunk 1 above)
Comment (enahum) [reply app/components/search_bar/search_box.js]: Yeah that change sounds good, also what about android?

Comment (sudheerDev) [reply app/components/search_bar/search_box.js]: There is no highlight for text so no changes are required for android.

Comment (sudheerDev) [reply app/components/search_bar/search_box.js]: Made changes to renaming prop

**Comment by sudheerDev** [comment]
@enahum Yes, it is used in other places. I need to update them but this would require more work than i thought because view will not take styles like color and font-size and we use different font-sizes. I need to check all of them individually

**Comment by sudheerDev** [comment]
@enahum reverted prop name to be the same as most styles are to the input now. 
taking out background alone to be used on view and rest of styles to the input component. So it keeps consistency with the rest of the places where we use these.

**Comment by enahum** [comment]
@sudheerDev build failures

**Comment by sudheerDev** [comment]
@enahum Looking into it. But i don't see why they fail

**Comment by sudheerDev** [comment]
@saturninoabril Need a second look at this. Minor changes though

**Pull request merged**